### DISCUSSION TITLE
microsoft surface + canon eoc1100d

### DISCUSSION THREAD

**Discussion by Artcool43** [photobooth-app/photobooth-app#196] [Q&A]
Good afternoon. Colleagues, please tell me. Have any of you tried installing this software on a combination of microsoft surface + canon eoc1100d? Is this possible? Thanks for answers.

**Comment by mgineer85** [comment]
While digicamcontrol itself seems mostly unmaintained, I do no see any reason why not.
My dev computer is Windows platform, so when you made digicamcontrol work with your cam it should be fine 👍
